sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v2_7/q78.sf100/simplified.txt

```diff
@@ -17,36 +17,35 @@ TakeOrderedAndProject [ss_sold_year,ss_item_sk,ss_customer_sk,ss_qty,ss_wc,ss_sp
                                 Project [ss_item_sk,ss_customer_sk,ss_quantity,ss_wholesale_cost,ss_sales_price,d_year]
                                   BroadcastHashJoin [ss_sold_date_sk,d_date_sk]
                                     Project [ss_item_sk,ss_customer_sk,ss_quantity,ss_wholesale_cost,ss_sales_price,ss_sold_date_sk]
-                                      Filter [sr_ticket_number]
-                                        SortMergeJoin [ss_ticket_number,ss_item_sk,sr_ticket_number,sr_item_sk]
-                                          InputAdapter
-                                            WholeStageCodegen (2)
-                                              Sort [ss_ticket_number,ss_item_sk]
-                                                InputAdapter
-                                                  Exchange [ss_ticket_number,ss_item_sk] #2
-                                                    WholeStageCodegen (1)
-                                                      Filter [ss_item_sk,ss_customer_sk]
+                                      SortMergeJoin [ss_ticket_number,ss_item_sk,sr_ticket_number,sr_item_sk]
```

Comment:
Do you have the benchmark number if active this change?

Reply:
Add a local benchmark code and result in PR description.